Index: linux-6.1.y/drivers/net/usb/qmi_wwan.c
===================================================================
--- linux-6.1.y.orig/drivers/net/usb/qmi_wwan.c
+++ linux-6.1.y/drivers/net/usb/qmi_wwan.c
@@ -1097,6 +1097,7 @@ static const struct usb_device_id produc
 	{QMI_MATCH_FF_FF_FF(0x05c6, 0xf601)},	/* MeigLink SLM750 */
 
 	/* 3. Combined interface devices matching on interface number */
+	{QMI_FIXED_INTF(0x1c9e, 0x9b05, 4)},
 	{QMI_FIXED_INTF(0x0408, 0xea42, 4)},	/* Yota / Megafon M100-1 */
 	{QMI_FIXED_INTF(0x05c6, 0x6001, 3)},	/* 4G LTE usb-modem U901 */
 	{QMI_FIXED_INTF(0x05c6, 0x7000, 0)},
